sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,13 @@
+-- Update boss Toravon the Ice Watcher
+UPDATE `creature_template` SET `InhabitType`=11 WHERE `entry`=38461;
+UPDATE `creature_template` SET `ScriptName`='' WHERE `entry`=38461;
```

Comment:
These 2 lines can be merged since entry is the same, but I suppose it is purely cosmetic:
```sql
UPDATE `creature_template` SET `InhabitType`=11, `ScriptName`='' WHERE `entry`=38461;
```